executor/common_linux.h

```diff
@@ -2417,17 +2416,24 @@ static unsigned long fs_image_segment_check(unsigned long size, unsigned long ns
 		size = IMAGE_MAX_SIZE;
 	return size;
 }
-#endif
 
-#if SYZ_EXECUTOR || __NR_syz_read_part_table
-// syz_read_part_table(size intptr, nsegs len[segments], segments ptr[in, array[fs_image_segment]])
-static long syz_read_part_table(volatile unsigned long size, volatile unsigned long nsegs, volatile long segments)
+// Setup the loop device needed for mounting a filesystem image. Takes care of
+// creating and initializing the underlying file backing the loop device and
+// returns the fds to the file and device.
+// Returns 0 on success, -1 otherwise.
+static int setup_loop_device(long unsigned size, long unsigned nsegs, struct fs_image_segment* segs, const char* memfd_name, const char* loopname, int* _memfd, int* _loopfd)
```

Comment:
We don't have any other precedents of identifiers starting with underscores. Please keep the code consistent. Start it with p or end with _p or something similar.
One other option is to return loop fd as the function result (it's called setup_loop_device in the end).

Reply:
I think the 2 pointers are more intuitive since we must return both memfd and loop file descriptors. I'll update their names to maintain consistency with the rest of code.